testng-core/src/test/java/test/listeners/github1602/IssueTest.java

```diff
@@ -34,17 +34,22 @@ public Object[][] getData() {
             "AfterInvocation_testMethod_SUCCESS",
             "BeforeInvocation_afterMethod_STARTED",
             "AfterInvocation_afterMethod_SUCCESS");
+
     List<String> baseList =
         Arrays.asList(
             "BeforeInvocation_beforeMethod_STARTED",
             "AfterInvocation_beforeMethod_FAILURE",
             "BeforeInvocation_testMethod_SKIP",
-            "AfterInvocation_testMethod_SKIP",
-            "BeforeInvocation_afterMethod_STARTED");
```

Comment:
Why did we have to remove this line ?

Reply:
hello, I changed "BeforeInvocation_afterMethod_STARTED"); to skipList.add("BeforeInvocation_afterMethod_SKIP");

in skiplist and failList there is different. 

    List<String> skipList = Lists.newArrayList(baseList);
    skipList.add("BeforeInvocation_afterMethod_SKIP");
    skipList.add("AfterInvocation_afterMethod_SKIP");

    List<String> failList = Lists.newArrayList(baseList);
    failList.add("BeforeInvocation_afterMethod_STARTED");
    failList.add("AfterInvocation_afterMethod_FAILURE");